.github/workflows/release-deploy-all.yml

```diff
@@ -88,7 +88,6 @@ env:
   QUEUE_PREFIX: ${{ secrets.QUEUE_PREFIX }}
   API_PORT: ${{ secrets.API_PORT }}
   WEB_PORT: ${{ secrets.WEB_PORT }}
-  DATABASE_NAME_KEY: ${{ secrets.DATABASE_NAME_KEY }}
```

Comment:
Not part of the PR. I see that the build namespace and keycloak realm are hard coded here. I believe we will be changing it to read from github secrets in future.

![image](https://github.com/bcgov/SIMS/assets/54600590/fead158c-3b69-4841-a920-d622868d36ed)

Reply:
We have the `BUILD_NAMESPACE` already available and used, for instance, in .github\workflows\env-setup-build-forms-server.yml
We can do the same for the `KEYCLOAK_REALM`.
By the way, @dheepak-aot for these ones that were hard-code, we used the `vars`.

Reply:
I have added a repository variable and changed it.